src/screens/SignUp/index.tsx

```diff
@@ -92,7 +93,9 @@ function SignUp() {
         />
         <Button type="submit" label={t('SignUp:lblSignUp')} loading={isLoading} primary />
       </form>
-      <Button label={t('SignUp:lblLogin')} onClick={() => reset()} loading={false} />
+      <Link to="/">
+        <Button label={t('SignUp:lblLogin')} onClick={() => reset()} loading={false} />
```

Comment:
same

Reply:
The button component has been removed  :suspect: